example/src/Pagination.js

```diff
@@ -0,0 +1,115 @@
+import { getPagination } from '@kaliber/pagination'
+import styles from './Pagination.css'
+
+export function Pagination({ current, max, deriveUrl, onChange }) {
+
+  // Semantics: https://www.a11ymatters.com/pattern/pagination/
+  return (
+    <nav className={styles.component} aria-label='Pagination'>
+      <Arrow
+        page={current + 1}
+        onClick={() => handleClick(current + 1)}
+        layoutClassName={styles.next}
+        label={`Next page (page ${current + 1})`}
+        dataX='goto-next-page'
+        {...{ max, deriveUrl }}
+      >
+        &rarr;
```

Comment:
Ik ben niet zo fan van `&iets;`
```suggestion
        →
```